lib/amqp/application.ex

```diff
@@ -128,15 +128,15 @@ defmodule AMQP.Application do
       iex> {:ok, conn1} = AMQP.Application.get_connection(:business_report)
       iex> {:ok, conn2} = AMQP.Application.get_connection(:analytics)
 
-  The defaut name is :default so These two configurations are equivalent:
+  The default name is :default so These two configurations are equivalent:
 
       config :amqp, connection: []
       config :amqp, connections: [default: []]
 
   ## Configuration options
 
-  * `:retry_interval` - The retry interval in milliseconds when the connection is failed to open (default `5000`)
-  * `:url` - AMQP URI for the connection
+  * `:retry_interval` - The retry interval in milliseconds when the connection is failed to open (default `5000`);
+  * `:url` - AMQP URI for the connection.
```

Comment:
:+1: for the format consistency. let's format all bulleted lists with the following format:

* no ";" or "." at the end of item
* no "." at the end of last item either

so let's keep the format here and update other parts instead.

examples: [ecto](https://hexdocs.pm/ecto/Ecto.Query.html#join/5-options), [phoenix](https://hexdocs.pm/phoenix/Phoenix.Controller.html#render/3-arguments)

Reply:
>  for the format consistency. let's format all bulleted lists with the following format:
> 
> * no ";" or "." at the end of item
> * no "." at the end of last item either
> 
> so let's keep the format here and update other parts instead.
> 
> examples: [ecto](https://hexdocs.pm/ecto/Ecto.Query.html#join/5-options), [phoenix](https://hexdocs.pm/phoenix/Phoenix.Controller.html#render/3-arguments)

Done. Please check.